storage/rocksdb/ha_rocksdb.h

```diff
@@ -770,14 +780,14 @@ class ha_rocksdb : public my_core::handler {
   int get_pk_for_update(struct update_row_info *const row_info);
   int check_and_lock_unique_pk(const uint key_id,
                                const struct update_row_info &row_info,
-                               bool *const found)
+                               bool *const found, const bool skip_unique_check)
```

Comment:
still more odd use of const

Reply:
It looks fine for me and keeping it will make a diff to FB smaller.